classes/Infrastructure/Auth/SentinelAccountManagement.php

```diff
@@ -34,7 +34,7 @@ public function findById(int $userId): UserInterface
     {
         $user = $this->sentinel->getUserRepository()->findById($userId);
 
-        if ($user instanceof SentinelUserInterface) {
+        if ($user instanceof Users\UserInterface) {
```

Comment:
The alias was actually more readable to me, tbh. 😕

Reply:
How is this more readable?